tests/bgp/conftest.py

```diff
@@ -708,6 +708,84 @@ def set_timeout_for_bgpmon(duthost):
         MAX_TIME_FOR_BGPMON = plt_reboot_ctrl.get('timeout', 180)
 
 
+@pytest.fixture(scope='session')
+def gather_info(tbinfo, duthost, nbrhosts):
```

Comment:
Actually, I missed the problem here. Don't use duthost since it may get you a supervisor. Use ` enum_rand_one_per_hwsku_frontend_hostname` and 'duthosts' to get the duthost here.

Reply:
I have changed this.